templates/base/src/pages/_app.js

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import { ApolloProvider } from '@apollo/react-hooks';
+import ReactQuerySample from '../reactQuery';
```

Comment:
We should do this in a independent module. Just like Smart Context. I don't want to write this code in _app.js because then it becomes difficult for me to maintain.